extension/lib/shopify.api.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @param {object} config
+ * @return {{}}
+ */
+module.exports = function (config) {
```

Comment:
This should be a class instead, which is initialized with the config in the constructor.

Reply:
This is the same structure like we are using at shopify-cart and shopify-user, therefor i just copied it from there. Are you sure we should change it just here and keep it at the other extensions? It's a little bit confusing i guess